.github/workflows/publish-next-package.yml

```diff
@@ -1,11 +1,11 @@
-name: Publish / Package
```

Comment:
I think these are mean to be two separate files, they shouldn't be merged into one